app/test_main.py

```diff
@@ -1,3 +1,67 @@
-from app.main import get_human_age
+import pytest
+from app import main
 
-# write your code here
+
+# Перевірка загальних тестів для кішки та собаки
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (35, 35, [4, 4]),  # Перевірка для 35 років
+        (100, 100, [21, 17]),  # Перевірка для 100 років
```

Comment:
Consider adding test cases for edge situations such as negative numbers, zero, or very large numbers beyond the provided examples. This will ensure that your tests cover all potential edge cases as required by the checklist.